src/Microsoft.PowerShell.Commands.Management/commands/management/Service.cs

```diff
@@ -1684,22 +1682,14 @@ protected override void ProcessRecord()
                             || (ServiceStartMode)(-1) != StartupType)
                         {
                             DWORD dwStartType = NativeMethods.SERVICE_NO_CHANGE;
```

Comment:
I can make that change for New-Service, but for Set-Service, I need to use dwStartType on line 1697 so it has to be initialized.

Reply:
Thanks for clafiry. I think we should leave as is.
Closed.

Reply:
In `New-Service`, dwStartType is also used in line 2045 when calling `NativeMethods.CreateServiceW`.
It turns out the following code works:
```
static void Blah ()
{
    if (!int.TryParse("1", out int result))
    {
        Console.WriteLine("Failed");
    }

    Console.WriteLine(result);
}
```

Reply:
In `New-Service`, dwStartType is also used in line 2045 when calling `NativeMethods.CreateServiceW`.
It turns out the following code works:
```
static void Blah ()
{
    if (!int.TryParse("1", out int result))
    {
        Console.WriteLine("Failed");
    }

    Console.WriteLine(result);
}
```
It would be the best if we can do it in a consistent way in those 2 places.

Reply:
Addressed